apple-silicon-support/packages/asahi-audio/default.nix

```diff
@@ -7,13 +7,13 @@ stdenv.mkDerivation rec {
   pname = "asahi-audio";
   # tracking: https://src.fedoraproject.org/rpms/asahi-audio
   # note: ensure that the providedConfigFiles list below is current!
-  version = "1.6";
+  version = "2.1";
```

Comment:
it seems that `2.2` got released in the meantime https://github.com/AsahiLinux/asahi-audio/tree/v2.2

Reply:
```diff
-  version = "2.1";
+  version = "2.2";
 
   src = fetchFromGitHub {
     owner = "AsahiLinux";
     repo = "asahi-audio";
     rev = "v${version}";
-    hash = "sha256-ZyA3MjlP1FV822P0AjYAk2YgNKfiEVYnBlPagvFOm+M=";
+    hash = "sha256-5YBQibt/dfJb9/TzF6rczeQE3ySm0SeewhZrgublu2E=";
   };
```

seems to be working too for me